geomesa-accumulo/geomesa-accumulo-tools/pom.xml

```diff
@@ -142,6 +142,34 @@
         </resources>
         <plugins>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.5.0</version>
```

Comment:
i'd move this up to the parent pom as a managed version number and remove the version here